scripts/Ubuntu22/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM ubuntu:jammy
+
+Maintainer Alexia Arthur arthur14@purdue.edu
```

Comment:
There is the [LABEL tag](https://docs.docker.com/engine/reference/builder/#maintainer-deprecated)